app/views/session_timeout/_expired.html.slim

```diff
@@ -5,4 +5,4 @@
         = image_tag(asset_url('clock.svg'), class: 'modal-ico')
         h3.mt0.mb2 = t('headings.session_timeout_warning')
         p.mb3 = t('session_expired_html', link: link_to(t('session_expired_link'), '.'))
```

Comment:
My bad. I didn't catch this during review. This link also needs to point to `request.original_url`.